packages/core/src/browser/style/tree.css

```diff
@@ -14,6 +14,10 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+:root {
+    --theia-DescriptionFocused:#fff;
```

Comment:
@kaiyue0329 we should use theming variables since they will be updated based on themes and not harcoded.
I believe we have a color already:

```ts
list.activeSelectionForeground
```